daemon/recording.c

```diff
@@ -402,7 +402,23 @@ void recording_start_daemon(call_t *call) {
 }
 // lock must be held
 void recording_start(call_t *call) {
+	bool was_paused = call->recording && !CALL_ISSET(call, RECORDING_ON);
 	CALL_SET(call, RECORDING_ON);
+
+	// Write resume timestamp to metadata file if resuming a paused recording
+	if (was_paused && call->recording->pcap.meta_fp) {
+		time_t now = rtpe_now / 1000000;
+		char timebuffer[32];
+		struct tm timeinfo;
+		if (localtime_r(&now, &timeinfo) != NULL) {
+			strftime(timebuffer, sizeof(timebuffer), "%FT%T", &timeinfo);
+			fprintf(call->recording->pcap.meta_fp,
+				"\nRecording resumed at: %s (%.3lf ms)\n",
+				timebuffer, rtpe_now / 1000.0);
+			fflush(call->recording->pcap.meta_fp);
```

Comment:
`fflush` isn't really needed since the file is unbuffered already, but can leave it if you prefer